src/ILCompiler/src/Program.cs

```diff
@@ -311,8 +311,7 @@ private int Run(string[] args)
             // Initialize type system context
             //
 
-            SharedGenericsMode genericsMode = _useSharedGenerics || !_isWasmCodegen ?
-                SharedGenericsMode.CanonicalReferenceTypes : SharedGenericsMode.Disabled;
+            SharedGenericsMode genericsMode = SharedGenericsMode.CanonicalReferenceTypes;
```

Comment:
`_useSharedGenerics` is now unused. Let's delete the command line option and field.

Reply:
Done.